packages/editor-ui/src/components/ParameterOptions.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="parameter-options" v-if="displayOptionsComputed">
```

Comment:
```suggestion
	<div class="parameter-options">
```

this should be moved up into parent component